src/doubleTrouble.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/**
+ * Implement doubleTrouble function:
+ *
+ * Function takes an array of integers (array), and a target (target), you must
+ * find out if any two consecutive numbers in the array sum to target. If so,
+ * remove the second number.
+ *
+ * Example:
+ *
+ * const array = [1, 2, 3, 4, 5]
+ * const target = 3
+ *
+ * 1+2 = target, so remove 2. No other pairs = target, so rest of array remains:
```

Comment:
@ivadimko what is implemented in tests?
1) there is no information about the position to start from
If we start checking from the end it with be quite another result [1, 2, 1]
2) should we recheck after a removing a number?

Reply:
@mgrinko line below
`Work through the array from left to right.`

another part of the description:
`if any two consecutive numbers in the array sum to target`

So we start at 0, move through the array and check `next number` + `current number`. If `next + current = target` we remove `next`.

Reply:
https://www.codewars.com/kata/double-trouble

Reply:
done in another PR without approving because it's minor changes